Pcap++/header/PcapLiveDevice.h

```diff
@@ -174,6 +174,12 @@ namespace pcpp
 			 */
 			int packetBufferSize;
 
+            /**
+             * Set the direction for capture packets. You can read more here:
+             * https://www.tcpdump.org/manpages/pcap.3pcap.html.
+             */
+            pcap_direction_t direction;
```

Comment:
Please use identation which is similar to the rest of the files (tabs in this case)

Reply:
I will do it.